tools/src/bin/build-blame.rs

```diff
@@ -254,18 +254,23 @@ fn build_blame_tree(
     git_repo: &git2::Repository,
     commit: &git2::Commit,
     tree_at_path: &git2::Tree,
+    parent_trees: Vec<Option<git2::Tree>>,
```

Comment:
You don't seem to be moving or mutating it so maybe just `&[Option<git2::Tree>]`?